### PR TITLE
Add Project BUILD

### PR DESCRIPTION
**Pull request by cgrail**

### CODE REVIEW

**Comment by cgrail** [comment]
@agraebe @feesch @DavidJones02 @devinea @brian-murray35 please review

**Comment by DavidJones02** [comment]
:+1:

File: src/projects/projects.json [hunk 1]
```diff
@@ -1,41 +1,331 @@
...
-  {"categories": [{"name": "general"},{"name": "data"},{"name": "featured"}], "projectTitle": "SAP Jam IT Project Work Patterns w/ JIRA", "projectDescription": "A fully-functioning example integration between SAP Jam and Atlassian JIRA. Learn how you can gather experts and collaborate on a complex IT project or troubleshoot critical escalations. View and manage JIRA issues within SAP Jam. Leverage the SAP Jam feed from JIRA.", "linkToGithub": "https://github.com/SAP/SAPJamWorkPatternJIRAIntegration", "linkToProject": "http://sap.github.io/SAPJamWorkPatternJIRAIntegration"},
-  {"categories": [{"name": "general"},{"name": "data"}], "projectTitle": "SHINE-SAP HANA Demo Content", "projectDescription": "SHINE is an education content to learn, develop and deploy SAP HANA Application Services (XS Engine) based applications within the SAP HANA Platform", "linkToGithub": "https://github.com/SAP/hana-shine"}
-]
+[{
+    "categories": [{
+        "name": "interface"
+    }, {
+        "name": "mobile"
```
Comment (DavidJones02) [inline src/projects/projects.json]: Please update to "cloud"

**Comment by agraebe** [comment]
@cgrail thanks for the contribution!

**Pull request merged**